primitives/core/src/lib.rs

```diff
@@ -108,8 +108,11 @@ impl ExecutionContext {
 		match self {
 			Importing | Syncing | BlockConstruction =>
 				offchain::Capabilities::none(),
-			// Enable keystore by default for offchain calls. CC @bkchr
-			OffchainCall(None) => [offchain::Capability::Keystore][..].into(),
+			// Enable keystore and transaction pool by default for offchain calls.
+			OffchainCall(None) => [
+				offchain::Capability::Keystore,
+				offchain::Capability::TransactionPool,
```

Comment:
@tomusdrw Does this make sense? It would be hard for me to specify my own context in GRANDPA code since it requires providing `Externalities` (I think it would defeat the purpose of using offchain for this).